cookbook/docs/ml_training.rst

```diff
@@ -1,6 +1,7 @@
 ################
 ML Training
 ################
+This section deals with training various Machine Learning models using different datasets. In the process of training, data is fed to the Machine Learning algorithm. The model learns from this data, i.e captures patterns which can be used to get predictions on new data.
```

Comment:
```suggestion
Understand how ML models can be trained from within Flyte, with an added advantage of orchestration benefits.
```

Reply:
Can you add "Explore how features can be engineered with the power of Flyte." to feature engineering README and "Flyte very much supports running your bioinformatics applications. Dive deeper into how that can be done." to the bioinformatics README as separate paragraphs?